### PR TITLE
Auditor portal new permissions

### PR DESCRIPTION
**Pull request by roman-karpovich**
More flexible permissions system without clear separation to groups. Added ability to assign existing unicef users to audits.

### CODE REVIEW

**Comment by ntrncic** [comment]
is this ready?

**Comment by ntrncic** [comment]
@roman-karpovich please resolve conflicts

**Comment by roman-karpovich** [comment]
@ntrncic in general, this branch is ready for review.

File: EquiTrack/audit/signals.py [hunk 1]
```diff
@@ -20,11 +19,9 @@ def create_user_receiver(instance, created, **kwargs):
 def staff_member_changed(sender, instance, action, reverse, pk_set, *args, **kwargs):
     if action == 'post_add':
         new_members = AuditorStaffMember.objects.filter(id__in=pk_set).select_related('user', 'user__profile')
-        for member in new_members:
-            engagement = Engagement.objects.get_subclass(id=instance.id)
-            if member.engagement_set.count() == 1:
-                send_invite_email(member, engagement)
 
+        engagement = Engagement.objects.get_subclass(id=instance.id)
```
[the remark this answers is not in the document]
Comment (roman-karpovich) [reply EquiTrack/audit/signals.py]: It returns correct subclass (Audit/SpotCheck/etc) with corresponding fields.
http://django-model-utils.readthedocs.io/en/latest/managers.html#inheritancemanager

**Comment by ntrncic** [comment]
@roman-karpovich could you please add comments to the code. It will make things easier to read and understand what's happening in the future.

**Comment by roman-karpovich** [comment]
@ntrncic added some comments to the code. please let me know if i'd forgot about something.

**Comment by robertavram** [comment]
merging for now... eventually we need to move to one consistent permissions solution across all modules.

**Comment by roman-karpovich** [comment]
@robertavram sure. action points module is building with new permissions. also we are working on migrating tpm, so old module is going to be removed in nearest future.

**Pull request merged**